packages/ref-data-retrievers/txc-retriever/index.py

```diff
@@ -184,11 +184,12 @@ def main(event, context):
             upload_tnds_data_to_s3()
 
     except Exception as e:
-        ssm_client.put_parameter(
-            Name="/scheduled/disable-table-renamer",
-            Value="true",
-            Type="String",
-            Overwrite=True
-        )
+        if os.getenv("STAGE") is not None:
+            ssm_client.put_parameter(
+                Name="/scheduled/disable-table-renamer-" + os.getenv("STAGE"),
+                Value="true",
+                Type="String",
+                Overwrite=True
+            )
```

Comment:
This change would need to be updated in all other files where the ssm param is referred